Tests/DashboardTest.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using HotelPriceScout.Data.Model;
+using HotelPriceScout.Pages;
+using HotelPriceScout.Data.Interface;
+
+
+
+namespace Tests
+{
+    public class DashboardTest
+    {
+        [Fact]
+        public void Test_If_CreateMonth_Creates_Correct_Month()
+        {
+            //Arrange and Act
+            Dashboard dashboard = new Dashboard();
+
+            dashboard.CreateMonth();
+            
+            //Assert
+            Assert.Equal(DateTime.Now.Month, dashboard.Month);
+
+        }
+
+        [Theory]
+        [InlineData("low", 100, 1)]
+        [InlineData("high", 1, 100)]
+        [InlineData("", 0, 0)]
+
+        public void Test_If_ChangeTextColorBasedOnMargin_Returns_Correct_Value_Low(string expected, int marketprice, int kompasPrice)
+        {
+            //Arrange and Act
+            Dashboard dashboard = new Dashboard();
+            string dash = dashboard.ChangeTextColorBasedOnMargin(marketprice, kompasPrice);
+            //Assert
+            Assert.Equal(expected, dash);
+
+        }
+
+        [Theory]
+        [InlineData("oi oi-caret-top", 100, 1)]
+        [InlineData("oi oi-caret-bottom", 1, 100)]
+        [InlineData("oi oi-minus", 0, 0)]
+        public void Test_If_ArrowDecider_Returns_Correct_Value(string expected, int marketprice, int kompasPrice)
+        {
+            //Arrange and Act
+            Dashboard dashboard = new Dashboard();
+            string dash = dashboard.ArrowDecider(marketprice, kompasPrice);
+            //Assert
+            Assert.Equal(expected, dash);
+        }
+
+        public static readonly object[][] ShowMoreInfoData =
```

Comment:
Skal static data ikke definineres som private members i toppen af test casen?

Reply:
MemberData kan kun reference public members. Men har sat den i toppen for at gøre det mere overskueligt 👍